daemon/media_socket.c

```diff
@@ -1434,7 +1462,6 @@ struct stream_fd *stream_fd_new(socket_t *fd, struct call *call, const struct lo
 	sfd->call = obj_get(call);
 	sfd->local_intf = lif;
 	g_queue_push_tail(&call->stream_fds, sfd); /* hand over ref */
-	g_slice_free1(sizeof(*fd), fd); /* moved into sfd, thus free */
```

Comment:
This looks like a mistake from my rebase onto the latest master. We originally wrote this recording stuff a few months ago. I will fix it.